QuickTests/FunctionalTests/DescribeTests.swift

```diff
@@ -0,0 +1,20 @@
+import XCTest
+@testable import Quick
+import Nimble
+
+class DescribeTests: XCTestCase {
+    
+    func testDescribeErrorsIfExampleGroupsAreNotInstantiated() {
+        
+        // fiddling with global state in this test to test a singleton, what could go wrong?
+        let exampleGroup = World.sharedWorld.currentExampleGroup
+        World.sharedWorld.currentExampleGroup = nil
```

Comment:
Yeah... it might be worth it to strip the last commit that adds the unit test from this PR, and I'll just merge the first two commits. We can then create an issue to track the unit test! :+1: